src/adapters/index.ts

```
@@ -48,7 +48,10 @@ const loadAdapters = async (context: QueryStringContext): Promise<void> => {
         return;
       }
       import("./ecommerce").then(({ default: load }): Promise<void> => load(tracker));
-      logger.warn(`No event specified, Loading ${context.environment}`);
+      const envDisplay = Array.isArray(context.environment) 
+        ? context.environment.join(', ') 
+        : context.environment;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)